configs/config.d/50-stash-origin-authz.cfg

```diff
@@ -32,12 +32,22 @@ if named stash-origin-auth
 
    sec.protbind * gsi
 
-   acc.authdb /etc/xrootd/Authfile
+   if defined STASH_ORIGIN_AUTHFILE
```

Comment:
Should `STASH_ORIGIN_AUTHFILE` for `stash-origin-auth` here be a different variable than the one used below for `stash-origin`?